site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -101,14 +101,6 @@ kubectl get secret allocator-client-ca -o json -n agones-system | jq '.data["cli
 
 The last command creates a new entry in the secret data map called `client_trial.crt` for `allocator-client-ca` and stores it. You can also achieve this by `kubectl edit secret allocator-client-ca -n agones-system`, and then add the entry.
 
-## Restart pods
```

Comment:
Thanks for the change. Please release guard it by 
```
{{% feature expiryVersion="1.7.0" %}}
## Restart pods
...
{{% /feature %}}
```

Reply:
Oops, my bad. Guarded it instead.